app/assets/stylesheets/main.scss

```diff
@@ -169,6 +169,10 @@ height:49px;
     margin-top: 5px;
     text-align: justify;
     box-shadow: unset;
+    &.w660-h350 {
```

Comment:
we`ve discussed this style of naming with Vitalii before - I do not prefer this, but if you agreed it in your team, go ahead.